### ISSUE TITLE
anagest allows intersecting links

### ISSUE THREAD

**Issue by MJochim** [IPS-LMU/EMU-webApp#233]
When inserting a set of gesture landmark events using `anagest`, one is asked which parent item the landmark events should be linked to. The parent level is defined in DBconfig, but within the parent level, every item is offered as a choice. This does not respect already existing links.

Ex: The `Word` level contains the items `it is futile`. Users can annotate a gesture and link it to `is`, and then create another gesture (later on the time axis) and link it to `it`. This results in illegal intersecting links.

**Comment by MJochim** [comment]
@raphywink Would you say this is critical for the next release?

**Comment by raphywink** [comment]
Not really... we should check for this in future but for now we just have to rely on the user not to do "silly" things like that :-). You should already have a function in the hierarchy modal code to check for valid links though right? In future we could probably just pass in the links to that function to check if they are valid or not...
